myapp/src/Day2/Que12/VoteReducer.jsx

```diff
@@ -0,0 +1,58 @@
+// 12.Create a simple voting application using the useReducer hook.
+// Set up a reducer to manage the votes for different options.
+// Create buttons representing voting options.
+// Use the useReducer hook to access the votes state and dispatch actions.
+// Allow users to vote for their preferred options.
+// Display the current vote count for each option.
+
+
+import React, { useState, useReducer } from "react";
+
+const voteReducer = (state, action) => {
+    //console.log(state[1]);
+  switch (action.type) {
+    
+    case "increment":
```

Comment:
@anantmishra-successive since increment is the only operation in this case, it is meaningless to use it as action. A better alternative to use as action could be "vote4BJP/vote4Congress". This way we can also avoid complicating the action entity by making it a string instead of object.